pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -120,6 +122,11 @@ public HelixBrokerStarter(String brokerHost, String helixClusterName, String zkS
     _helixExternalViewBasedRouting = new HelixExternalViewBasedRouting(_propertyStore, _spectatorHelixManager,
         pinotHelixProperties.subset(ROUTING_TABLE_PARAMS_SUBSET_KEY));
     _tableQueryQuotaManager = new TableQueryQuotaManager(_spectatorHelixManager);
+    _tableSchemaCache = new TableSchemaCache(_propertyStore,
+        _pinotHelixProperties.getInt(CommonConstants.Broker.CONFIG_OF_TABLE_SCHEMA_MAXIMUM_CACHE_SIZE,
```

Comment:
none of these configs are useful. If we go beyond cache size, we will be hitting ZK for every call